src/components/Payment/index.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import creditcard from '../../assets/creditcard.png'
+import date from '../../assets/date.png'
+import coin from '../../assets/coins.png'
+import locked from '../../assets/locked.png'
+
+import {
+  Input
+} from 'reactstrap';
+
+
+class Payment extends React.PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      card_number: '',
+      no_space_card_number: '',
+      expiry_date: '',
+      amount: '',
+      amount_without_comma: '',
+      cvs: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
+  }
+
+
+  handleChange(e) {
+    let val = e.target.value.replace(/\D/g, '');
+    switch (e.target.id) {
+      case 'card_number':
+        let addSpace = val.replace(/(.{4})/g, '$1 ').trim();
+        let noSpace = addSpace.replace(/ /g, '');
+        this.setState(
+          {
+            card_number: addSpace,
+            no_space_card_number: noSpace
+          }
+        )
+        break;
+      case 'expiry_date':
+        let slashed = val.replace(/(.{2})/, '$1/').trim();
+        this.setState(
+          {
+            expiry_date: slashed,
+          }
+        )
+        break;
+      case 'amount':
+        let comma = val.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+        this.setState(
+          {
+            amount: comma,
+            amount_without_comma: val
+          }
+        )
+        break;
+      default:
+        this.setState(
+          {
+            [e.target.id]: val
+          }
+        )
+        break;
+    }
+  }
+
+
+  render() {
+    // let info = this.props.info;
+    let { card_number, expiry_date, cvs, amount} = this.state
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'card_number' is not in camel case.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)